src/Controller/Component/AuthComponent.php

```diff
@@ -170,21 +172,11 @@ class AuthComponent extends Component
     protected $_authorizeObjects = [];
 
     /**
-     * The session key name where the record of the current user is stored. Default
-     * key is "Auth.User". If you are using only stateless authenticators set this
-     * to false to ensure session is not started.
+     * Storage object.
      *
-     * @var string
-     */
-    public $sessionKey = 'Auth.User';
```

Comment:
We could, though getting isn't the real issue, what to do if user sets it is :smile:

Should the SessionStorage config be updated if $sessionKey is set to new value?